src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_TicketingStack/Ticketing_TicketTabNavStack/TicketTabNav_PurchaseTabScreen/Components/FareProducts/FareProductTile.tsx

```diff
@@ -7,11 +7,31 @@ import {
   FareContractTexts,
   TicketingTexts,
   useTranslation,
+  TranslateFunction,
 } from '@atb/translations';
-import {getStaticColor, StaticColor} from '@atb/theme/colors';
-import {TransportModes} from '@atb/components/transportation-modes';
+import {
+  getStaticColor,
+  getTransportationColor,
+  StaticColor,
+  TransportColor,
+} from '@atb/theme/colors';
+
 import {FareProductTypeConfig} from '@atb/configuration';
 import {useTextForLanguage} from '@atb/translations/utils';
+import {
+  TransportMode,
+  TransportSubmode,
+} from '@atb/api/types/generated/journey_planner_v3_types';
+
+// eslint-disable-next-line no-restricted-imports
+import {TransportModePair} from '@atb/components/transportation-modes/TransportModes';
+
+type TransportModesType = {
```

Comment:
This type could probably also live in `transportation-modes/TransportModes.tsx`, alongside `TransportModePair`? I know it's only used once now, but moving it there will hopefully prevent someone else from creating it again somewhere else next time.

Reply:
Yes. After using src/utils/use-transportation-color.ts though, this will be completely removed instead